databook/_toc.yml

```diff
@@ -11,79 +11,95 @@ chapters:
-    - file: visual-behavior/VBO-ExperimentData
-      title: Ophys Experiment Data
-    - file: visual-behavior/VBO-Tutorial-Compare_trial_types
-      title: Tutorial Comparing Trial Types
-    - file: visual-behavior/VBO-Tutorial-Downloading_data_and_exploring_the_manifest
-      title: Tutorial Downloading Data
-  - file: visual-behavior/VB-Neuropixels
-    title: Visual Behavior Neuropixels
+    - file: physiology/ephys/visual-coding/vcnp
+      sections:
+      - file: physiology/ephys/visual-coding/vcnp-quickstart
+      - file: physiology/ephys/visual-coding/vcnp-data-access
+      - file: physiology/ephys/visual-coding/vcnp-units
+      - file: physiology/ephys/visual-coding/vcnp-receptive-fields
+      - file: physiology/ephys/visual-coding/vcnp-lfp
+      - file: physiology/ephys/visual-coding/vcnp-optotagging
+      - file: physiology/ephys/visual-coding/vcnp-session
+      - file: physiology/ephys/visual-coding/vcnp-cheatsheet
+      - file: physiology/ephys/visual-coding/vcnp-links
+    - file: physiology/ephys/visual-behavior/VB-Neuropixels
+      title: Visual Behavior Neuropixels
+      sections:
+      - file: physiology/visual-behavior/VBN-Dataset
+      - file: physiology/visual-behavior/VBN-SessionData
+      - file: physiology/visual-behavior/VBN-Tutorial-Aligning_Neural_Data_to_Stimuli.ipynb
+        title: Tutorial Aligning Neural Data to Stimuli
+      - file: physiology/visual-behavior/VBN-Tutorial-Aligning_Behavior_Data_to_Task_Events.ipynb
+        title: Tutorial Aligning Behavioral Data to Task Events
+      - file: physiology/visual-behavior/VBN-Tutorial-Analyzing_LFP_Data.ipynb
+        title: Tutorial Analyzing LFP data
+    - file: physiology/ephys/cell-type-lookup-table/ctlut-background
+      sections:
+      - file: physiology/ephys/cell-type-lookup-table/ctlut-session-data
+      - file: physiology/ephys/cell-type-lookup-table/ctlut-optotagging
+      - file: physiology/ephys/cell-type-lookup-table/ctlut-accessing-data
+      - file: physiology/ephys/cell-type-lookup-table/ctlut-identifying-tagged-units
+    - file: physiology/ephys/np-ultra/np-ultra                    # Also a placeholder that needs actual content
+  - file: physiology/stimuli/stimuli
     sections:
-    - file: visual-behavior/VBN-Dataset
-    - file: visual-behavior/VBN-SessionData
-    - file: visual-behavior/VBN-Tutorial-Aligning_Neural_Data_to_Stimuli.ipynb
-      title: Tutorial Aligning Neural Data to Stimuli
-    - file: visual-behavior/VBN-Tutorial-Aligning_Behavior_Data_to_Task_Events.ipynb
-      title: Tutorial Aligning Behavioral Data to Task Events
-    - file: visual-behavior/VBN-Tutorial-Analyzing_LFP_Data.ipynb
-      title: Tutorial Analyzing LFP data
-- file: microns-em/em-background
+    - file: physiology/stimuli/passive-visual-stimuli/visual-stimuli-list                    # Also a placeholder that needs actual content
+      sections:
+      - physiology/stimuli/passive-visual-stimuli/visual-stimuli-np
```

Comment:
no, there is one page for passive visual stimuli and one page for behavior task.